src/archivematicaCommon/tests/test_execute_functions.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: UTF-8 -*-
+
+import executeOrRunSubProcess as execsub
+
+
+def test_capture_output():
+    """Tests behaviour of capture_output when executing sub processes."""
+
+    # Test that stdout and stderr are not captured by default
```

Comment:
Regarding the earlier comment about `True`/`False` defaults - if `capture_output` is `True` by default for `launchSubProcess` then these tests will need updating - one to test the `True` default, and one passing `False`.

Reply:
As above, I think the correct approach is for the default value to be `False`, but I will add an additional test to confirm that ommitting the parameter results in **no output**, and that specifying `True` and `False` also produces the right result.